src/pages/api/ngo.ts

```diff
@@ -72,8 +66,7 @@ const getNgoData = async (): Promise<Fuse<CachedData>> => {
   const cacheExpireTime = Date.now() - maxCacheAgeInMinutes * 60 * 1000
 
   if (!cache.time || cache.time < cacheExpireTime) {
```

Comment:
Cachovacia logika je zbytocna ked je to lokalny json subor. Toto malo zmysel ked sme robili requesty na stranku financnej spravy a parsovali HTML.

Reply:
Ono toto bol quick fix, cize mozno by som si to tu nechal.